doc/translation_process.md

```diff
@@ -66,7 +66,7 @@ username = USERNAME
 
 **For Windows**
```

Comment:
Since we have zero maintainers on Windows, I think this can be safely removed. More so the that the link to the transifex install instructions is identical to the linux one.

Reply:
Windows section from doc removed